tests/test_driver/firebase_storage/reference_e2e.dart

```diff
@@ -341,9 +341,9 @@ void setupReferenceTests() {
         // });
       },
       // putFile is not supported in web.
-      // iOS works locally but times out on CI. We ought to check this periodically
+      // iOS & macOS work locally but times out on CI. We ought to check this periodically
       // as it may be OS version specific.
```

Comment:
Should we add a TODO or FIXME in order to keep track of those?

Reply:
@Lyokone Sure. I've added to this issue: https://github.com/firebase/flutterfire/issues/9375

Reply:
Thanks :)